web/src/message_live_update.js

```diff
@@ -52,10 +52,10 @@ export function update_starred_view(message_id, new_value) {
         const $elt = $row.find(".star");
         const $star_container = $row.find(".star_container");
         if (starred) {
-            $elt.addClass("fa-star").removeClass("fa-star-o");
+            $elt.addClass("zulip-icon-star-filled").removeClass("zulip-icon-star");
             $star_container.removeClass("empty-star");
         } else {
-            $elt.removeClass("fa-star").addClass("fa-star-o");
+            $elt.removeClass("zulip-icon-star-filled").addClass("zulip-icon-star");
```

Comment:
We probably need to change these icons in the /help/ articles; `git grep fa-star` will help find them.

Reply:
Sure! I'll do that.

Reply:
Fixed, and added before/after screenshots to the PR descrip.